src/main/java/com/team2502/robot2020/subsystem/ShooterSubsystem.java

```diff
@@ -0,0 +1,58 @@
+package com.team2502.robot2020.subsystem;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel;
+import com.team2502.robot2020.Constants;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+// CANSparkMax methods, http://www.revrobotics.com/content/sw/max/sw-docs/java/com/revrobotics/CANSparkMax.html
+
+
+public class ShooterSubsystem extends SubsystemBase {
+
+    private final CANSparkMax shootRight;
+    private final CANSparkMax shootLeft;
+
+
+
+    public boolean shooterRunning;
+
+    public ShooterSubsystem() {
+        //Fix int for Motor ID
+        shootLeft = new CANSparkMax(Constants.RobotMap.Motors.SHOOTER_LEFT, CANSparkMaxLowLevel.MotorType.kBrushless);
+        shootRight = new CANSparkMax(Constants.RobotMap.Motors.SHOOTER_RIGHT, CANSparkMaxLowLevel.MotorType.kBrushless);
+
+        // Add copy motor, motor follows other.
+        //FOllOWS?
+        //shootLeft.follow(shootRight);
+
+    }
+
+    //Sets Shooters speed between -1.0 and 1.0
```

Comment:
ALL of these pre-method comments should be javadoc comments

[what is a javadoc?](https://team-2502.github.io/programming/Making-a-JavaDoc-for-Your-Special-Code.html)